File: app/src/main/java/com/mikhailovskii/trakttv/data/entity/MovieTrack.java
package com.mikhailovskii.trakttv.data.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class MovieTrack {

    @SerializedName("movie")
    public Movie movie;
    @SerializedName("watchers")
    @Expose
    private int watchersNumber;

    public int getWatchersNumber() {
        return watchersNumber;
    }

    public void setWatchersNumber(int watchersNumber) {
        this.watchersNumber = watchersNumber;
    }

}

File: app/src/main/java/com/mikhailovskii/trakttv/data/entity/MovieTrack.kt
package com.mikhailovskii.trakttv.data.entity

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class MovieTrack {

    @SerializedName("movie")
    var movie: Movie? = null
    @SerializedName("watchers")
    @Expose
    var watchersNumber: Int = 0

}
